test/verify/machineslib.py

```diff
@@ -37,6 +37,29 @@ def readFile(name):
     return content
 
 
+# Preparations for physical disk storage pool
+def prepareDisk(m):
+    m.add_disk("50M", serial="DISK1")
+    wait(lambda: m.execute("ls -l /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_DISK1"))
+    m.execute("parted /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_DISK1 mklabel gpt")
+
+
+# Preparations for iscsi storage pool
+def prepareSCSIDisk(m, target_iqn):
```

Comment:
This is not a SCSI disk that you are preparing here , the name is again misleading. You are just setting up the ISCSI server and creating some LUNS there. This is not a disk.
Please rename it to prepareStorageDeviceOnISCSI

Reply:
Done